index.html

```diff
@@ -27,17 +27,119 @@ <h1>I'm Saningo Glad to see you</h1>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-2"></div>
+              <div class="card-desc">
+                <h2>Muilti-Post Stories</h2>
+                <ul>
+                  <li>FACEBOOK</li>
+                  <li>Full stack Dev</li>
+                  <li>2023</li>
+                </ul>
+                <div class="desc">
+                  <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                  <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                  <button type="button">See Project</button>
+                </div>
+              </div>
+          </div>
 
-          <div class="empty">
+          <div class="card">
+            <div class="card-3"></div>
+              <div class="card-desc">
+                  <div>
+                    <h2>Facebook 360</h2>
+                    <ul>
+                      <li>FACEBOOK</li>
+                      <li>Full stack DEV</li>
+                      <li>2023</li>
+                    </ul>
+                    <div class="desc">
+                      <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                      <button type="button">See Project</button>
+                    </div>
+                  </div>
+              </div>
+          </div>
 
+          <div class="card">
+            <div class="card-4"></div>
+            <div class="card-desc">
+                <div>
+                  <h2>Facebook React</h2>
+                  <ul>
+                    <li>FACEBOOK</li>
+                    <li>Full stack DEV</li>
+                    <li>2023</li>
+                  </ul>
+                  <div class="desc">
+                    <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button type="button">See Project</button>
+                  </div>
+                </div>
+            </div>
           </div>
+
+        </section>
+        <!-- About Me section -->
+        <section class="about-me">
+          <h1>About Me</h1>
+          <p>Hello I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, dont hestatite to contact me.</p>
+           <button type="button">Get my Resume</button>
```

Comment:
- [ ] The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design). Kindly add this effect on the button.
![nynrtb](https://user-images.githubusercontent.com/26767383/153623680-2764f973-5858-4b07-a62f-ecea1dcf280e.PNG)

Reply:
I have added the hover effect and made the buttons active

Reply:
- [ ] Kindly fix the hover and do it as previous reviewer ask you to it following Figma design. Here you can see your button's hover: 
 
![image](https://user-images.githubusercontent.com/74506933/153671188-01fb4e10-0b8a-47e5-b9a4-a1cb61d1b311.png)